app/views/form.html

```diff
@@ -46,14 +46,11 @@
     </keyword-input>
   </div>
 
-  <h3>Droits attendus</h3>
+  <h3>Valeurs attendues</h3>
 
-  <p>
-    Attention : veillez à supprimer ici les droits que vous ne testez pas.
-    <br>Veillez aussi à ajouter les droits dont vous testez que le montant calculé doit être nul (en mettant "0" comme valeur attendue).
```

Comment:
The only problematic case would be if :
- The user wants to make sure a value is null
- When s·he creates the test, mes-aides is right, and the value is null
- s·he doesn't explicitly set it to null
- Later there is a regression and the result becomes non-null. It is not detected.

This doesn't seem to be so likely (if user want to test a specific value, I guess they'll think about setting it to 0). So ok for removing the text.